frappe/printing/doctype/print_format/print_format.js

```diff
@@ -35,13 +35,20 @@ frappe.ui.form.on("Print Format", {
 			else if (frm.doc.custom_format && !frm.doc.raw_printing) {
 				frm.set_df_property("html", "reqd", 1);
 			}
-			frm.add_custom_button(__("Make Default"), function () {
-				frappe.call({
-					method: "frappe.printing.doctype.print_format.print_format.make_default",
-					args: {
-						name: frm.doc.name
-					}
-				})
+			frappe.db.get_value('DocType', frm.doc.doc_type, ['default_print_format', 'custom'], (r) => {
+				if (r.default_print_format != frm.doc.name && r.custom) {
```

Comment:
@anupamvs @surajshetty3416 should we show the button anyway and improve the validation message?
"Standard DocType cannot have default print format, use Customize Form"
We can link to Customize Form here.